charts/vpa.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: gardener-dashboard-vpa
+  namespace: garden
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind:       Deployment
+    name:       gardener-dashboard
+  updatePolicy:
+    updateMode: "Off"
```

```suggestion
    updateMode: "Off"

```
add newline at the end?